homeassistant/util/__init__.py

```diff
@@ -116,6 +121,38 @@ def get_random_string(length=10):
     return ''.join(generator.choice(source_chars) for _ in range(length))
 
 
+def load_json(filename: str) -> Union[List, Dict]:
+    """Load JSON data from a file and return as dict.
+
+    Defaults to returning empty dict (if e.g. file not found).
+    """
+    try:
+        with open(filename, encoding='utf-8') as fdesc:
+            return json.loads(fdesc.read())
+    except FileNotFoundError:
+        _LOGGER.debug('JSON file not found: %s', filename)
+    except ValueError:
+        _LOGGER.warning('Could not parse JSON content: %s', filename)
```

Comment:
This error is pretty bad, we should probably just blow up?

Reply:
You are right. I didn't want to change the behavior much from what was currently in place/use.
I'll raise a HomeAsssistantError, analogous to load_yaml, instead.

Reply:
This means that we do no longer support empty files?